academic-services/knowledge-exploration-service/tutorial-schema-design.md

```diff
@@ -0,0 +1,385 @@
+            "AuthorName": "chiyuan huang",
+            "OriginalAffiliationName": "Microsoft Research, Redmond, WA, 98052, USA",
+            "AffiliationName": "microsoft"
+        },
+        {
+            "Sequence": 4,
+            "OriginalAuthorName": "Chieh-Han Wu",
+            "AuthorName": "chiehhan wu",
+            "OriginalAffiliationName": "Microsoft Research, Redmond, WA, 98052, USA",
+            "AffiliationName": "microsoft"
+        },
+        {
+            "Sequence": 5,
+            "OriginalAuthorName": "Yuxiao Dong",
+            "AuthorName": "yuxiao dong",
+            "OriginalAffiliationName": "Microsoft Research, Redmond, WA, 98052, USA",
+            "AffiliationName": "microsoft"
+        },
+        {
+            "Sequence": 6,
+            "OriginalAuthorName": "Anshul Kanakia",
+            "AuthorName": "anshul kanakia",
+            "OriginalAffiliationName": "Microsoft Research, Redmond, WA, 98052, USA",
+            "AffiliationName": "microsoft"
+        }
+    ],
+    "OriginalAbstract": "An ongoing project explores the extent to which artificial intelligence (AI), specifically in the areas of natural language processing and semantic reasoning, can be exploited to facilitate the stu...",
+    "Title": "microsoft academic graph when experts are not enough",
+    "TitleWords": [ "microsoft", "academic", "graph", "when", "experts", "are", "not", "enough"],
+    "AbstractWords": [ "an", "ongoing", "project", "explores", "the", "extent", "to", "which", "artificial", "intelligence", "ai", "specifically", "in", "areas", "of", "natural", "language", "processing", "and", "semantic", "reasoning", "can", "be", "exploited","facilitate", "stu"]
+}
+```
+
+After inspecting the data, we can now create the schema for it.
+
+### Display only attributes
+
+The display attrbitues are attributes that are needed for display only, such as **OriginalTitle**, **OriginalAuthorName** In this case, we can define them as **Blob** type such that KES won't create any index for them.
+
+The following schema elements reflect the display attributes:
+
+```json
+{
+  "attributes": [
+    {"name": "OriginalTitle","type": "blob?"},
+    {"name": "OriginalAbstract","type": "blob?"},
+    
+    {"name": "VenueFullName","type": "blob?"},
+
+    {"name": "AuthorAffiliations","type": "Composite*"},
+    {"name": "AuthorAffiliations.OriginalAuthorName","type": "blob?"},
+    {"name": "AuthorAffiliations.OriginalAffiliationName","type": "blob?"},
+
+    {"name": "FieldsOfStudy","type": "Composite*"},
+    {"name": "FieldsOfStudy.OriginalName","type": "blob?"},
+  ]
+}
+```
+
+### Filter attributes
+
+Filter attributes are attributes that can be used to filter entity data.
+
+The numeric attributes that we want to filter by would be the publication's **Year, CitationCount, EstimatedCitationCount**. Depending on the filter UX we want to provide, we can add **equals** and/or **is_between** operations to the filterable numeric attributes.
```

Comment:
>UX [](start = 149, length = 2)

should this be functionality?  Are we restricting user to a UI presentation of filters?

Reply:
Changed to "Operational Attributes" and yes, we are designing the schema based on the UI presentation of filters.